### PR TITLE
Add gridifier@2.0.3 w/ git auto-update

### LINKED ISSUE #10706: [Request] Add gridifier
**Library name:**: gridifier

**Git repository url:**: https://github.com/nix23/gridifier 

**npm package name or url** (if there is one): https://www.npmjs.com/package/gridifier

**License (List them all if it's multiple):** 
Creative Commons BY-NC-SA 4.0.
developer license

**Official homepage:**: http://gridifier.io/

=====================

Notes from cdnjs maintainer:
Please read the [README.md](https://github.com/cdnjs/cdnjs#cdnjs-library-repository) and [CONTRIBUTING.md](https://github.com/cdnjs/cdnjs/blob/master/.github/CONTRIBUTING.md) document first.

We encourage you to add a library via sending pull request,
it'll be faster than just opening a request issue,
since there are tons of issues, please wait with patience,
and please don't forget to read the guidelines for contributing, thanks!!

### PR DESCRIPTION
**Pull request by dakshshah96**
close #10706, cc @nix23

Pull request for issue: #10706 

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/nix23/gridifier
 * Official website (optional, not the repository): http://gridifier.io/
 * NPM package url (optional): https://www.npmjs.com/package/gridifier
 * License and its reference: CC BY-NC-SA 4.0
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 15
   - Count of stars: 301
   - Count of forks: 17
 * NPM download stats (optional):
   - Downloads in the last day: 3
   - Downloads in the last week: 9
   - Downloads in the last month: 42

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/gridifier/package.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  "keywords": [
+    "grids",
+    "async-grids",
+    "grid-layouts"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/nix23/gridifier"
+  },
+  "license": "CC BY-NC-SA 4.0",
```
Comment (pvnr0082t) [inline ajax/libs/gridifier/package.json]: We should follow the SPDX format. You can take a look at [license-list.json](https://github.com/cdnjs/cdnjs/blob/master/tools/license-list.json#L79) under `tools` directory.

File: ajax/libs/gridifier/package.json [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  "author": "Eduard Pleh <@nix23>",
+  "autoupdate": {
+    "source": "git",
+    "target": "https://github.com/nix23/gridifier.git",
+    "fileMap": [
+      {
+        "basePath": "dist",
+        "files": [
+          "gridifier?(.min).js"
```
Comment (pvnr0082t) [inline ajax/libs/gridifier/package.json]: We should use black list instead of white list. Let's use `"**/*.!(*gz)"` here.

Comment (PeterDaveHello) [inline ajax/libs/gridifier/package.json]: I think we prefer to use `"**/!(*.gz)"`?

**Comment by dakshshah96** [comment]
@PeterDaveHello Fixed the files. The problem was because I copied and pasted contents of the file instead of using the file directly. 😄

**Comment by PeterDaveHello** [comment]
@dakshshah96 oh that's a bad idea to copy paste the content, please don't do that anymore, thanks.

**Comment by dakshshah96** [comment]
@PeterDaveHello Yes, learning with each pull request. Thanks to you! 😄

**Comment by PeterDaveHello** [comment]
@dakshshah96 there are some files in the old versions should be excluded in this case, you can try it next time.

**Pull request merged**